server/handlers.js

```diff
@@ -110,19 +112,30 @@ const readHandler = async (req, res) => {
     const resData = []
     await client.connect()
     const db = client.db('pang');
```

Comment:
you're connecting twice. I assume you're connecting to mongo everytime for each handler. a good practice would be to connect only once when you start the server and share the connection over the different handlers as it avoids recreating connexion each time (which consume ressources) 

If you look for `share mongo connexion accross node app` you should be able to find some example or pseudo code to help you.